diff --git a/CMakeLists.txt b/CMakeLists.txt
index ed38c3b..27b50f2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -68,7 +68,7 @@ message(STATUS "Building ${PROJECT_NAME} with Qt ${Qt5Core_VERSION}")
 if (QTXDG_INSTALL_DEFAPPS_CONFIG)
     file(GLOB QTXDG_CONFIG_FILES config/*.conf)
     set(QTXDG_DEFAPPS_CONF_INSTALL_DIR
-        "${LXQT_ETC_XDG_DIR}" CACHE PATH "Path to qtxdg default apps conf files install dir")
+        "/etc/xdg" CACHE PATH "Path to qtxdg default apps conf files install dir")
     mark_as_advanced(QTXDG_DEFAPPS_CONF_INSTALL_DIR)
 endif()
 
@@ -168,7 +168,7 @@ install(EXPORT
 
 if (QTXDG_INSTALL_DEFAPPS_CONFIG)
     install(FILES ${QTXDG_CONFIG_FILES}
-        DESTINATION "${QTXDG_DEFAPPS_CONF_INSTALL_DIR}"
+        DESTINATION "etc/xdg"
         COMPONENT Runtime
     )
 endif()
diff --git a/src/xdgiconloader/CMakeLists.txt b/src/xdgiconloader/CMakeLists.txt
index 1269bd7..cf7bdd0 100644
--- a/src/xdgiconloader/CMakeLists.txt
+++ b/src/xdgiconloader/CMakeLists.txt
@@ -9,10 +9,12 @@ set(xdgiconloader_PRIVATE_H_FILES
 
 set(xdgiconloader_CPP_FILES
     xdgiconloader.cpp
+    ../qtxdg/xdgdirs.cpp
 )
 
 set(xdgiconloader_PRIVATE_INSTALLABLE_H_FILES
     xdgiconloader_p.h
+    ../qtxdg/xdgdirs.h
 )
 
 
diff --git a/src/xdgiconloader/xdgiconloader.cpp b/src/xdgiconloader/xdgiconloader.cpp
index 91515ca..41d5aa2 100644
--- a/src/xdgiconloader/xdgiconloader.cpp
+++ b/src/xdgiconloader/xdgiconloader.cpp
@@ -56,6 +56,7 @@
 #include <QSvgRenderer>
 
 #include <private/qhexstring_p.h>
+#include "../qtxdg/xdgdirs.h"
 
 //QT_BEGIN_NAMESPACE
 
@@ -574,7 +575,7 @@ QThemeIconInfo XdgIconLoader::loadIcon(const QString &name) const
             const auto unthemedInfo = unthemedFallback(name, QIcon::themeSearchPaths());
             if (unthemedInfo.entries.isEmpty()) {
                 /* Freedesktop standard says to look in /usr/share/pixmaps last */
-                const QStringList pixmapPath = (QStringList() << QString::fromLatin1("/usr/share/pixmaps"));
+                const QStringList pixmapPath = XdgDirs::dataDirs(QString::fromLatin1("/pixmaps"));
                 const auto pixmapInfo = unthemedFallback(name, pixmapPath);
                 if (pixmapInfo.entries.isEmpty()) {
                     return QThemeIconInfo();
