From 0d45fef77bbbd53619784cf13f9b222f1e003a33 Mon Sep 17 00:00:00 2001
From: Alex ter Weele <alex.ter.weele@gmail.com>
Date: Sun, 9 Sep 2018 16:22:22 -0500
Subject: [PATCH] Do not use command -p in Makefile.

---
 Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 49c7bf0..fe9f0f3 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -304,9 +304,9 @@ LIBTARSNAP_A=	lib/libtarsnap.a
 # access post-configure info: lib-platform/platform.h, config.h, and
 # -DHAVE_CONFIG_H.
 apisupport-config.h:
-	( export CC="${CC}"; export CFLAGS="-I${top_srcdir}/lib-platform -I${builddir} ${DEFS} ${CFLAGS}"; command -p sh $(srcdir)/libcperciva/apisupport/Build/apisupport.sh "$$PATH") > apisupport-config.h.tmp && command -p mv apisupport-config.h.tmp apisupport-config.h
+	( export CC="${CC}"; export CFLAGS="-I${top_srcdir}/lib-platform -I${builddir} ${DEFS} ${CFLAGS}"; sh $(srcdir)/libcperciva/apisupport/Build/apisupport.sh "$$PATH") > apisupport-config.h.tmp && mv apisupport-config.h.tmp apisupport-config.h
 cpusupport-config.h:
-	( export CC="${CC}"; export CFLAGS="${CFLAGS}"; command -p sh $(srcdir)/libcperciva/cpusupport/Build/cpusupport.sh "$$PATH" ) > cpusupport-config.h.tmp && command -p mv cpusupport-config.h.tmp cpusupport-config.h
+	( export CC="${CC}"; export CFLAGS="${CFLAGS}"; sh $(srcdir)/libcperciva/cpusupport/Build/cpusupport.sh "$$PATH" ) > cpusupport-config.h.tmp && mv cpusupport-config.h.tmp cpusupport-config.h

 # Add libcperciva files to dist
 #
--
2.18.0
